src/Makefile

```diff
@@ -10,7 +10,11 @@ CC ?= gcc
 SRCDIR = $(shell find . -type d -not -regex './obj.*' -printf '%P ')
 OBJDIR = $(patsubst %,obj/%,$(SRCDIR))
 
-LUASRC = $(shell find . -regex '[^\#]*\.lua' -printf '%P ')
+GENLUASRC := core/version.lua
+LUASRC = $(shell for f in $(GENLUASRC); do \
+	           if test ! -f $$f; then echo -n "$$f "; fi; \
```

Comment:
Request: Eliminate the special logic for building even when the `core.version` module is missing. Simpler to make sure this module always exists. Otherwise code that uses `core.version` will always need to check if it is there or not.